_custom_build/backend.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import build_meta as _orig
```

Comment:
maybe add a link to https://setuptools.pypa.io/en/latest/build_meta.html#dynamic-build-dependencies-and-other-build-meta-tweaks at the very top of this file (e.g. in the module-level docstring), for our future selves when they stumble on this

Reply:
Thank you for reviewing! Great idea, I did that now.

By the way, would you mind if I implemented a similar solution for https://github.com/googlefonts/compreffor?

Reply:
yes please, thanks